src/App.tsx

```diff
@@ -2,24 +1,48 @@
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42).map(n => `Item ${n}`);
+const allItems = getNumbers(1, 42).map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [amounts, setAmount] = useState<number>(5);
+  const [pages, setPage] = useState<number>(1);
+
+  const getVisibleItem = (items: string[], amount: number, page: number) => {
+    const start = (page - 1) * amount;
+
+    return items.slice(start, start + amount);
+  };
+
+  const visibleItem = getVisibleItem(allItems, amounts, pages);
```

Comment:
Ensure that the `Pagination` component correctly handles the `onPageChange` callback to update the `pages` state when the page changes.